src/components/core/PivotTable.tsx

```diff
@@ -866,6 +891,32 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
         );
     }
 
+    private setGroupingProvider(sortedByFirstAttr: boolean) {
+        this.groupingProvider = GroupingProviderFactory.createProvider(sortedByFirstAttr);
+    }
+
+    private sortedByFirstAttribute(columnDefs: ColDef[]) {
```

Comment:
return type definition

Reply:
also I would say as this is a condition check... it would be better to start with `is`

`isSortedByFirstAttibute`